plugin/ale/cursor.vim

```diff
@@ -7,6 +7,20 @@ endif
 
 let g:loaded_ale_cursor = 1
 
+" Return a formatted message according to g:ale_echo_msg_format variable
+function! s:GetMessage(linter, type, text) abort
+    let msg = g:ale_echo_msg_format
+    let linter = a:linter
```

Comment:
Ah ok, the linter variable is there for debugging purpose and as usually I forgot to remove it.